deploy/runtime/defaults/PV Battery_Third Party.txt

```diff
@@ -887,6 +887,12 @@ batt_inverter_efficiency_cutoff
 1
 1
 90
+batt_life_model
+1
+1
+1
+1
+1
```

Comment:
Just checking is the default lifetime model supposed to be 1 for "NMC Gr Life Model" for all configs except Fuel Cell?